switchsdk/src/main/java/net/gini/switchsdk/SwitchSdk.java

```diff
@@ -542,6 +543,28 @@ public SwitchSdk setReviewTitleText(@StringRes final int text) {
         return this;
     }
 
+    /**
+     * Skip the "Analyzing Complete" screen (the popup that is shown after all the extractions
+     * have been found). This screen should be only skipped if you have a custom version of this
+     * screen.
+     *
+     * @return the instance of the available SDK
+     */
+    public SwitchSdk skipAnalyzingCompletedScreen() {
+        mSkipAnalyzingScreen = true;
+        return this;
+    }
+
+    /**
+     * Returns whether the "Analyzing Complete" screen (shown after all the extractions have been
+     * found) should be skipped.
+     *
+     * @return whether the analyzing complete screen should be skipped
+     */
+    boolean shouldSkipAnalyzingCompletedScreen() {
```

Comment:
I'm not sure if people would have about getting that value. They either leave the default or set their own value. I cannot think of a use case where they need to read it. But we can leave it for consistency.

Reply:
The getter is package private - only used inside the SDK code.